lib/messages/abstract-issue.js

```diff
@@ -70,7 +70,9 @@ class AbstractIssue extends Message {
     };
     if (this.major) {
       if (this.abstractIssue.body_html) {
-        core.text = mrkdwn(this.abstractIssue.body_html);
+        const { text, image } = mrkdwn(this.abstractIssue.body_html);
+        core.text = text;
+        core.image_url = image;
```

Comment:
do we need a conditional here in case there is no image? I suppose Slack is fine with us passing `image_url: undefined`?